README.md

```diff
@@ -1,181 +1,115 @@
 # Contour
-Contour is a typesafe, Kotlin-first API for complex layouts on Android.
 
-## What Contour Is Not
-There is a lot of buzz and interest around writing views in code right now with the development of [Jetpack Compose](https://developer.android.com/jetpack/compose) & [Anko](https://github.com/Kotlin/anko).
+<a href="https://www.youtube.com/watch?v=PLpzbLgHCLU"><img src="screenshots/droidcon_talk_cover.png"></a>
```

Comment:
I would prefer this after the first paragraph. On mobile, you'll want the text to be first rather than a huge image.

Reply:
Done